cmd/juno/juno.go

```diff
@@ -145,6 +147,7 @@ const (
 	colourUsage                           = "Use `--colour=false` command to disable colourized outputs (ANSI Escape Codes)."
 	ethNodeUsage                          = "WebSocket endpoint of the Ethereum node. To verify the correctness of the L2 chain, " +
 		"Juno must connect to an Ethereum node and parse events in the Starknet contract."
+	noEthNodeUsage           = "Disables the connection to the Ethereum node."
```

Comment:
The description should be: "L1 verification is disabled since an Ethereum node is not provided."